tests/api/test_account.py

```diff
@@ -577,29 +577,32 @@ def test_user_with_cancelled_fulfillments(
     assert fulfillments[1]["status"] == FulfillmentStatus.CANCELED.upper()
 
 
-ACCOUNT_REGISTER_MUTATION = """
-    mutation RegisterAccount(
-        $password: String!,
-        $email: String!,
-        $redirectUrl: String!
-    ) {
-        accountRegister(
-            input: {
-                password: $password,
-                email: $email,
-                redirectUrl: $redirectUrl
-            }
-        ) {
-            errors {
-                field
-                message
-            }
-            user {
-                id
-            }
-        }
-    }
-"""
+def _build_account_register_query(requires_redirect=True):
```

Comment:
I'm not sure if it's a good solution. We should build a query without any conditions. Instead of condition, you can create mutation with optional `$redirectUrl: String`, and everything should work in both cases with `redirectUrl` inside query variables and without it.